lib/presentation/widgets/movie_list_presentation/grid_widgets/row_movie_preview_grid.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+
+import '../../../../core/utils/constants/dimens.dart';
+import 'text_title_grid.dart';
+
+class RowMoviePreviewGrid extends StatelessWidget {
```

Comment:
little change here to GridRowMoviePreview for better readability , same for the file name